pages/index.js

```diff
@@ -9,7 +9,7 @@ import { Footer } from '../components/footer';
 import styles from '../styles/Home.module.css';
 import TransactionChart from '../components/transaction-chart';
 import TransactionOperationModal from '@components/transaction-operation-modal';
```

Comment:
A request, can you change this to ../components/transaction-operation-modal :)

Reply:
ok, Will be done